src/components/Settings/Password.vue

```diff
@@ -10,24 +10,122 @@
 			/>
 		</template>
 		<template v-slot:content>
-			<p>Clicking this button will send you a verification...</p>
+			<p>
+				Clicking this button will send you a verification email.
+				As a final step in this process, you'll need to click the
+				link in that email to successfully update your account password.
+			</p>
+			<kv-alert
+				v-if="isPasswordRequestSuccess"
+				class="alert"
+				role="alert"
+				variant="success"
+				:can-close="true"
+			>
+				<b>{{ successMsg }}</b>
+			</kv-alert>
+			<kv-alert
+				v-if="isPasswordRequestFailure"
+				class="alert"
+				role="alert"
+				variant="danger"
+				:can-close="true"
+			>
+				Error sending password
```

Comment:
This should say something about a password reset email, not just a password, as we aren't actually sending them a password. Maybe "There was a problem sending the password reset email"?